### DISCUSSION TITLE
zotero 7.0 will be supported?

### DISCUSSION THREAD

**Discussion by henusys** [retorquere/zotero-better-bibtex#2458] [Q&A] [answered]
When will zotero 7.0 be supported? Thank you very much.
zotero_7_for_developers: 
https://www.zotero.org/support/dev/zotero_7_for_developers

**Comment by retorquere** [comment, marked as answer]
BBT works on 6 and 7.

**Reply by henusys** [reply]
BBT is a great plugin that has helped me a lot academically. Thank you!

**Reply by earthmoon** [reply]
> BBT is a great plugin that has helped me a lot academically. Thank you!

I feel the same!

**Reply by haoming-li-ling** [reply]
Is there any news on this? Any timeline on when this plugin will be supported? I think the zotero beta 7 is already being pushed to those on zotero beta 6.

**Reply by retorquere** [reply]
No news yet. I'm slowly moving BBT to restartless in preparation, that's the first phase.

**Reply by FeralFlora** [reply]
Well, f*** me, I shouldn't have updated until BBT was ready for Zotero 7. Got too excited. My whole workflow depends on BBT.

**Reply by retorquere** [reply]
Downgrading is not possible?

**Reply by FeralFlora** [reply]
I am not sure, but I hope so. Some form of downgrading should be possible, but I don't know how seamless it will be. It depends on whether and how the database is changed.

**Reply by FeralFlora** [reply]
Downgrading is possible:
>That said, Zotero 7 doesn't currently make any database upgrades, so you should be able to switch back and forth between Zotero 6 and 7 without problems.
https://forums.zotero.org/discussion/105094/announcing-the-zotero-7-beta

**Reply by jangid19** [reply]
Thank you for all your hard work. BBT elevates Zotero above all other reference managers.

**Comment by satikusala** [comment]
Looking forward the update when it is ready.

**Reply by retorquere** [reply]
It's slow going. BBT is a complex plugin, and Zotero 7 changes some fairly fundamental things.

**Reply by TomBirkland** [reply]
Please know we really appreciate all your hard work on this. BBT is a key part of our workflows, and makes Zotero so much more useful. I will test the 7.0 beta as soon as BBT is ready!

**Comment by Russ-dh** [comment]
Idem dito here. BBT is a lifesaver for working with Obsidian. But I am leery of using betas, and so am waiting for the availability in a "stable" version of Zotero before upgrading to the new version of the software.

**Reply by retorquere** [reply]
I don't follow. BBT supports both 6 and 7, and 7 is no longer beta.

**Reply by Russ-dh** [reply]
My apologies. I thought that BBT doesn't yet fully work with seven. I had tried some weeks ago and got the message that BBT didn't support 7. But I didn't find a place where you unequivocally state that BBT now supports the release (not beta) version of 7. 
I'll install seven and have a look.
Off topic: other ways to make a single cash donation other than Paypal?

**Reply by retorquere** [reply]
BBT has had full support for Zotero 7 since somewhere mid-2023. #2522 is pinned to the top of the issue list and tracks the work zotero 7 compatibility. 

> Off topic: other ways to make a single cash donation other than Paypal?

Only other way I know only works if you live in the Netherlands.

**Reply by Russ-dh** [reply]
I live in the Hague... :) heb je het over een tikkie?

**Reply by retorquere** [reply]
Well that was unexpected... inderdaad, tikkie.
